### PR TITLE
remove assembly progress assertion

### PR DESCRIPTION
**Pull request by mifi**
Assemblies no longer seem to consistently emit progress

See failed test runs:
- https://github.com/transloadit/node-sdk/runs/5767629359?check_suite_focus=true
- https://github.com/transloadit/node-sdk/runs/5784606211?check_suite_focus=true
- https://github.com/transloadit/node-sdk/runs/5798550618?check_suite_focus=true
- https://github.com/transloadit/node-sdk/runs/5804663978?check_suite_focus=true

### CODE REVIEW

**Review by juliangruber — COMMENTED** [review]
Code lgtm, however conceptually I don't think this is right 🤔  Curious what Tim says

**Comment by mifi** [comment]
Yea, maybe I was a bit too quick to assume that this is happening due to some internal rewrite or debouncing/throttling of assembly progress events in the api

**Comment by tim-kos** [comment]
If anything our move to Assembly progress emitted by node should make progress event emits more consistent.

This is something we need to look into.

**Comment by mifi** [comment]
I'm not able to reproduce the issue locally. After looking more into this, I realised that it's not really assembly progress from the backend that's failing. Rather the problem lies in `waitForCompletion` and `awaitAssemblyCompletion` **inside this node sdk** which polls assembly status immediately after creating the assembly and (then every 1 sec). Basically I think the assemblies are finishing before the node-sdk even has the time to send its first `getAssembly` call, thus the node-sdk is not able to emit any progress. I don't see anything wrong with this, and I think it is indeed an unreasonable assumption and assertion that the assembly should not be finished within x milliseconds. So I think it's still the right thing to do to remove this assertion.

Basically my theory as to why this has started failing more frequently now, is because assemblies now finish more quickly than before, which is a good problem to have!

**Review by tim-kos — APPROVED** [review]
SGTM!
